fili-generic-example/src/main/java/com/yahoo/wiki/webservice/application/GenericMain.java

```diff
@@ -0,0 +1,120 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.wiki.webservice.application;
+
+import com.yahoo.bard.webservice.application.HealthCheckServletContextListener;
+import com.yahoo.bard.webservice.application.MetricServletContextListener;
+import com.yahoo.bard.webservice.data.config.dimension.DimensionConfig;
+
+import com.codahale.metrics.servlet.InstrumentedFilter;
+import com.codahale.metrics.servlets.AdminServlet;
+
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.eclipse.jetty.server.Handler;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.handler.HandlerList;
+import org.eclipse.jetty.server.handler.ResourceHandler;
+import org.eclipse.jetty.servlet.FilterHolder;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHolder;
+import org.glassfish.jersey.servlet.ServletContainer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.EnumSet;
+
+import javax.servlet.DispatcherType;
+
+/**
+ * Launch Bard in Embedded Jetty.
+ */
+public class GenericMain {
+    private static final Logger LOG = LoggerFactory.getLogger(GenericMain.class);
+
+    /**
+     * Makes the dimensions passthrough.
+     * <p>
+     * This method sends a lastUpdated date to each dimension in the dimension cache, allowing the health checks
+     * to pass without having to set up a proper dimension loader. For each dimension, d, the following query is
+     * sent to the /v1/cache/dimensions/d endpoint:
+     * {
+     *     "name": "d",
+     *     "lastUpdated": "2016-01-01"
+     * }
+     *
+     * @param port  The port through which we access the webservice
+     *
+     * @throws IOException If something goes terribly wrong when building the JSON or sending it
+     */
+    private static void markDimensionCacheHealthy(int port) throws IOException {
```

Comment:
Now that we're using the DruidDimensionLoader, we shouldn't need this method at all I think.

And if we yank this method, the `LOG` isn't used anymore either.